### ISSUE TITLE
Drawing tools focus ring varies with image size/scale

### ISSUE THREAD

**Issue by eatyourgreens** [zooniverse/front-end-monorepo#6072]
## Package
- [x] lib-classifier

## Describe the bug
When you tab through the drawn marks for a drawing task, the focus ring depends on the image size:

WildWatch Burrowing Owl (large-ish images) has a thin focus ring.
<img width="800" alt="Screenshot from WildWatch Burrowing Owl, showing a drawn rectangle with a thin, light blue focus ring." src="https://github.com/zooniverse/front-end-monorepo/assets/59547/ffaf449f-ee88-4352-876a-72f87e40b1eb">

I Fancy Cats (small images) has a large, distracting focus ring.
<img width=800 alt="Screenshot from I Fancy Cats, showing a drawn ellipse around a kitten's face, with a very thick focus ring." src="https://github.com/zooniverse/front-end-monorepo/assets/59547/ff013813-8992-4a50-a519-393c5529b967">

## To Reproduce
Draw some marks on a subject with a drawing task, then tab through the drawn marks. Try this for a variety of subject sizes.

## Expected behavior
There should be a thin focus ring, independent of the subject image size.
